plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -225,6 +224,28 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
             instancePolicyOrTemplate.setInstallGpuDrivers(true)
         }
 
+        final disk = task.config.getDiskResource()
+        if( disk )
+            instancePolicy.addDisks(
+                AllocationPolicy.AttachedDisk.newBuilder()
+                    .setNewDisk(
+                        AllocationPolicy.Disk.newBuilder()
+                            .setType(disk.type ?: 'pd-standard')
+                            .setSizeGb(disk.request.toGiga())
+                    )
+            )
+
+        else if( fusionEnabled() )
+            instancePolicy.addDisks(
+                AllocationPolicy.AttachedDisk.newBuilder()
+                    .setNewDisk(
+                        AllocationPolicy.Disk.newBuilder()
+                            .setType('local-ssd')
+                            .setSizeGb(375)
+                    )
+                    .setDeviceName('fusion')
```

Comment:
Here is where we give the mounted disk a name and then here 

https://github.com/nextflow-io/nextflow/blob/efedec747d46008ef466807962f8023b1c6fea92/plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchFusionAdapter.groovy#L49-L56

we mount it inside the container. For Fusion we always want to mount it as `/tmp` and maybe the same makes sense when you are not using Fusion, because at the end you want to use it as "scratch" and that means always `/tmp`.